### PR TITLE
Updating history per Feb 2022

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@inoxx03 Can you please review/approve this?

**Review by stoobie — DISMISSED** [review]
LGTM

File: HISTORY.md [hunk 1]
```diff
@@ -18,11 +20,26 @@ Review the history of significant updates to the _Red Hat supplementary style gu
...
-<!-- Updated as of December 14 2021 -->
+<!-- Updated as of February 23 2022 -->
+
+<a name="2022-february"></a>
+## February 2022
+
+* Added alternatives for the terms [master and slave](https://redhat-documentation.github.io/supplementary-style-guide/#conscious-language) in the conscious language section
+* Added a new section for guidelines specific to [cloud services documentation](https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services)
+* Added product-specific terminology for [Red Hat Directory Server](https://redhat-documentation.github.io/supplementary-style-guide/#_red_hat_directory_server)
+* Council decided to follow IBM Style Guide guidance on Java terms "throw" and "catch"
```
Comment (joaedwar) [inline HISTORY.md:31]:
```suggestion
Council decided to follow _IBM Style Guide_ guidance on Java terms "throw" and "catch"
```

**Review by joaedwar — APPROVED** [review]
lgtm

Just one suggestion

**Review by IngridT1 — DISMISSED** [review]
LGTM

File: HISTORY.md (hunk 1 above)
Comment (IngridT1) [reply HISTORY.md:31]: Good point. I missed this.

Comment (bergerhoffer) [reply HISTORY.md:31]: good catch thanks!

**Review by IngridT1 — APPROVED** [review]
LGTM

**Pull request merged**